main.tf

```diff
@@ -293,7 +332,7 @@ resource "google_monitoring_alert_policy" "errors" {
   conditions {
     display_name = "Error condition"
     condition_matched_log {
-      filter = "severity=ERROR AND resource.labels.service_name = ${module.google_cloud_run.service_name}"
+      filter = "severity=ERROR ${ var.create_job ? "" : "AND resource.labels.service_name = " + module.google_cloud_run.service_name}"
```

Comment:
Si create_job est true ça donne "severity=ERROR ${module.google_cloud_run.service_name}" sans le AND ?
Si false "severity=ERROR ${ AND resource.labels.service_name = module.google_cloud_run.service_name}"

Reply:
oui, c'est à améliorer, j'ai pas trouvé comment filtrer sur un job